relay-server/src/testutils.rs

```diff
@@ -114,10 +114,18 @@ pub fn create_test_processor(config: Config) -> EnvelopeProcessorService {
     #[cfg(feature = "processing")]
     let (_aggregator, _) = mock_service("aggregator", (), |&mut (), _| {});
 
+    #[cfg(feature = "processing")]
+    let redis = {
+        let url = std::env::var("RELAY_REDIS_URL")
+            .unwrap_or_else(|_| "redis://127.0.0.1:6379".to_owned());
```

Comment:
This seems a bit sketchy. 

Can the `redis` be just defaulted to `Option`?  If the config not provided, just make it `None` and/or also produce the error from inside of the tests, to make sure that it won't get un-notice if someone changes anything.

Reply:
I've now changed to what i think you had in mind